### PR TITLE
Add validate_setup script to make sure the server can send emails

### PR DESCRIPTION
**Pull request by leplatrem**
screenshot from 2017-04-11 16-55-59

### CODE REVIEW

File: validate_setup.py [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+    client.create_record(data={"script": "validate kinto-emailer setup"})
+
+    # 4. If signing is enabled for this collection, then see integration with kinto-signer.
+    signed_resources = capabilities.get("signer", {"resources": []})["resources"]
+    ids = [(r["source"]["bucket"], r["source"]["collection"]) for r in signed_resources]
+    signing_enabled = (args.bucket, args.collection) in ids
+
+    if signing_enabled:
+        # 4. Request a review.
+        client.patch_collection(data={"status": "to-review"})
```
Comment (Natim) [inline validate_setup.py]: If it is not enabled we can wait forever, because we won't receive anything.

Comment (Natim) [inline validate_setup.py]: Actually there are 2 hooks, one for the review and one for the record creation.

File: validate_setup.py [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+
+        if SUBJECT_EMAIL_CREATION not in subjects:
+            if time.time() - first_try < RETRY_TIMEOUT:
+                time.sleep(3)
+                sys.stdout.write('.')
+                sys.stdout.flush()
+                continue
+        break
+
+    if SUBJECT_EMAIL_CREATION in subjects:
```
Comment (Natim) [inline validate_setup.py]: Why is this not in the while?

File: validate_setup.py (hunk 1 above)
Comment (leplatrem) [inline validate_setup.py]: ?

File: validate_setup.py (hunk 2 above)
Comment (leplatrem) [inline validate_setup.py]: I thought it would be easier to read...

File: validate_setup.py (hunk 1 above)
Comment (Natim) [inline validate_setup.py]: Forget about it just talking to myself :)

**Comment by leplatrem** [comment]
@Natim do you approve it?
screenshot from 2017-04-12 15-18-09

**Pull request merged**